notebooks/vertex_genai/solutions/vertex_llm_tuning.ipynb

```diff
@@ -0,0 +1,1001 @@
+{
```

Comment:
<p><code>google-cloud-aiplatform</code> is included in the requirements.txt, but newer versions are required? If so, please add an information about the version requirement in markdown.</p><p>(Or I think it's ok to update the requirements.txt itself if it doesn't affect other dependencies.)</p><p> </p><p>And it seems <code>sequence-evaluate</code> is not very widely used? (6 stars.)</p><p>How about switching it to more popular one like Hugging Face <code>evaluate</code> library?</p><p> </p><p>https://github.com/huggingface/evaluate/tree/main</p><p> </p><p>https://medium.com/@sthanikamsanthosh1994/understanding-bleu-and-rouge-score-for-nlp-evaluation-1ab334ecadcb</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/395/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Good catch! Done!</p>